nvflare/app_common/aggregators/intime_accumulate_model_aggregator.py

```diff
@@ -196,6 +196,11 @@ def accept(self, shareable: Shareable, fl_ctx: FLContext) -> bool:
             return False
 
         contributor_name = shareable.get_peer_prop(key=ReservedKey.IDENTITY_NAME, default="?")
+
+        peer_ctx = fl_ctx.get_peer_context()
+        if contributor_name == "?" and peer_ctx:
+            contributor_name = peer_ctx.get_identity_name("?")
+
```

Comment:
the shareable IDENTITY_NAME prop is not set in the WFCommClient case, however the peer identity name can also be found in the peer context in this case

Reply:
instead of doing it here, maybe we want to somehow set it in upstream (WFCommClient) so any other aggregators that try to get this "IDENTITY_NAME" can get it?

Reply:
@SYangster where did we do this now?

Reply:
https://github.com/NVIDIA/NVFlare/pull/2530/files#diff-7207f57b6f393aefda5f21823d25b52ab07902f0257b80dd73f7ea86c0104f4fR163